ktor-client/ktor-client-plugins/ktor-client-content-negotiation/common/src/io/ktor/client/plugins/contentnegotiation/ContentNegotiation.kt

```diff
@@ -77,11 +92,77 @@ public class ContentNegotiation internal constructor(
             registrations.add(registration)
         }
 
+        /**
+         * Adds a type to the list of types that should be ignored by [ContentNegotiation].
+         *
+         * The list contains the [HttpStatusCode] type by default.
+         */
+        public inline fun <reified T> ignoreType() {
+            ignoredTypes.add(T::class)
+        }
+
+        /**
+         * Clear all configured ignored types including defaults.
```

Comment:
This api is not really usefull. If you want to have all default types, but not string, you need to copy the code from sources and keep it up to date.

Reply:
Let's add remove with type as well